src/components/auth/PrivacyAwareSignInButton.tsx

```
@@ -0,0 +1,122 @@
+}
+
+export function PrivacyAwareSignInButton({
+  children,
+  mode = 'modal',
+  forceRedirectUrl,
+  fallbackRedirectUrl,
+  signUpForceRedirectUrl,
+  signUpFallbackRedirectUrl,
+}: PrivacyAwareSignInButtonProps) {
+  const [showPrivacyConsent, setShowPrivacyConsent] = useState(false);
+  const { openSignIn, openSignUp } = useClerk();
```

ESLint

[error] 'openSignUp' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)